public/assets/elfrun.js

```diff
@@ -5,24 +5,30 @@ var readTest = [];
 var lastCharWritten = 0;
 
 // ASSUME GLOBAL ACCESS TO RISCV
-function elfRunNextInst() {
+function elfRunNextInst(command) {
     var instVal;
     var stopCount = 10000;
     var tryCount = 0;
 
+    if (command != null && command[0] === 'l') {
```

Comment:
you could write `if (command && command[0] === '1')` this will handle, also, the case when `command` is `undefined`